lineapy/plugins/argo_pipeline_writer.py

```diff
@@ -127,14 +132,14 @@ def _write_operators(
             WORFLOW_NAME=self.dag_config.get(
                 "workflow_name", self.pipeline_name.replace("_", "-")
             ),
-            IMAGE=self.dag_config.get("image", "argo_pipeline:latest"),
+            IMAGE=self.dag_config.get(
+                "image", f"{self.pipeline_name}:lineapy"
+            ),
             IMAGE_PULL_POLICY=self.dag_config.get(
                 "image_pull_policy", "Never"
             ),
             SERVICE_ACCOUNT=self.dag_config.get("service_account", "argo"),
-            KUBE_CONFIG=self.dag_config.get(
-                "kube_config", os.path.expanduser("~/.kube/config")
-            ),
+            KUBE_CONFIG=kube_config,
```

Comment:
i'd prefer a different key here ( instead of the original string path its now a union of str and "code string to get path"). not a showstopper though and leaving the final decision to you.

Reply:
using `KUBE_CONFIG_PATH_CODE_BLOCK` upon discussion.